test/e2e/reboot_test.go

```diff
@@ -0,0 +1,60 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+var _ = Describe("Nmstate network connnections persistency", func() {
+	var (
+		vlan101 = nmstatev1alpha1.State(`interfaces:
+interfaces:
```

Comment:
Help me here please. Having tests focused on a single thing is important. However, our main concern is whether the default interface comes up with the same IP, otherwise we lose access to the host. Could we test that specifically?

Reply:
(I want a test that fails due to changed IP and forces us to fix that bug)

Reply:
Well this is just a test to ensure that network configuration is apply before kubernetes-nmstate is up and ready, but we can increase the reboot test with the issue we have found at this PR or a follow up PR whatever you see it's better.